### PR TITLE
Feat/creator page

### PR DESCRIPTION
**Pull request by khubak**

### CODE REVIEW

File: src/app/lib/api/creator/queries.ts [hunk 1]
```diff
@@ -8,21 +8,19 @@ import { Nullable } from '@/models/common'
...
+  })
   return response.data
 }
 
 export const fetchCreators = async (params: CreatorParams): Promise<Creator[]> => {
   const { data } = await fetchWrapper<Creator[]>({
     params,
     path: `${CREATOR}/${GET}`,
-    revalidateCacheInSeconds: 15 * 60,
+    //revalidateCacheInSeconds: 15 * 60,
```
Comment (d-reader-luka) [inline src/app/lib/api/creator/queries.ts:23]: maybe put it as a 1 minute? thoughts?

File: src/components/comic/cards/DefaultCard.tsx [hunk 2]
```diff
@@ -46,15 +47,9 @@ export const DefaultComicCard: React.FC<Props> = ({ comic, className }) => {
...
       </div>
       <div className='flex flex-col px-2 pb-1 max-sm:gap-1'>
-        <Text
-          title={comic.title}
-          as='p'
-          styleVariant='body-normal'
-          fontWeight='bold'
-          className='line-clamp-1 overflow-ellipsis max-sm:text-sm'
-        >
+        <TextWithOverflow as='p' styleVariant='body-normal' fontWeight='bold' className='max-sm:text-sm'>
```
Comment (d-reader-luka) [inline src/components/comic/cards/DefaultCard.tsx:50]: This component is causing some issues on Comic Card. Check your version and https://www.dreader.app/

File: src/components/discover/ClearAllButton.tsx [hunk 3]
```diff
@@ -10,7 +11,7 @@ export const ClearAllButton: React.FC<ClearAllButtonProps> = ({ className }) =>
   const clearAll = useDiscoverQueryStore((state) => state.resetToDefaultInitState)
 
   return (
-    <Button onClick={clearAll} variant='secondary' size='md' className={className}>
+    <Button onClick={clearAll} variant='secondary' size='md' className={cn('h-10 min-w-[100px]', className)}>
```
Comment (d-reader-luka) [inline src/components/discover/ClearAllButton.tsx:14]: you should not update height
It should be 42px per figma if I'm not mistaken. That's a standard height for button that has normal size

File: src/components/discover/DiscoverSearchBar.tsx [hunk 4]
```diff
@@ -35,7 +35,7 @@ export const DiscoverSearchBar: React.FC<Props> = ({ className }) => {
       <Input
         placeholder={getPlaceholder()}
         value={searchTerm || ''}
-        className='pl-10 pr-10 w-full max-w-[100%] rounded-lg'
+        className='pl-10 pr-10 w-full max-w-[100%]'
```
Comment (d-reader-luka) [inline src/components/discover/DiscoverSearchBar.tsx:38]: Input should have same height as buttons on the left.
Also, please add debounce on search

File: src/components/shared/buttons/ShareButton.tsx [hunk 5]
```diff
@@ -27,5 +27,14 @@ export const ShareButton: React.FC<Props> = ({ title = '', text = '' }) => {
...
 
-  return <Button onClick={handleShare} icon={Share2} iconOnly variant='secondary' />
+  return (
+    <Button
+      onClick={handleShare}
+      size={'sm'}
+      icon={Share2}
+      iconOnly
+      variant='secondary'
+      className='h-[42px] rounded-[10px]'
```
Comment (d-reader-luka) [inline src/components/shared/buttons/ShareButton.tsx:37]: I'm repeating myself, there is no need to override class height and border.
We should use size='md' here and that's it, no need to override className

You can check that size='md' have the same specifications that you've sent within `className`.

File: src/components/comic/cards/DefaultCard.tsx (hunk 2 above)
Comment (khubak) [reply src/components/comic/cards/DefaultCard.tsx:50]: I've added w-full missing tag in DefaultComicCard

**Pull request merged**